chsdi/views/wmtscapabilities.py

```diff
@@ -5,10 +5,12 @@
 from gatilegrid import GeoadminTileGrid
 
 from chsdi.models.bod import get_wmts_models
-from chsdi.lib.helpers import sanitize_url
+from chsdi.lib.helpers import sanitize_url, is_valid_hostname
 from chsdi.lib.validation import MapNameValidation
 from chsdi.lib.filters import filter_by_geodata_staging, filter_by_map_name
 
+WMTS_DEFAULT_HOST = 'wmts.geo.admin.ch'
```

Comment:
https://github.com/geoadmin/mf-chsdi3/blob/master/rc_prod#L13

Here we have it already no?